snapcraft/plugins/make.py

```diff
@@ -25,6 +25,12 @@
 
 Additionally, this plugin uses the following plugin-specific keyword:
 
+    - artifacts:
+      (list)
+      Link/copy the given files from the make output to the snap
+      intstallation directory. If specified, the 'make install'
```

Comment:
Thanks Blake, fixed.

On Fri, Aug 12, 2016 at 8:25 AM, Blake Rouse <notifications@github.com>
wrote:

> In snapcraft/plugins/make.py
> <https://github.com/snapcore/snapcraft/pull/715#discussion_r74589626>:
>
> > @@ -25,6 +25,12 @@
> >
> >  Additionally, this plugin uses the following plugin-specific keyword:
> >
> > +    - artifacts:
> > +      (list)
> > +      Link/copy the given files from the make output to the snap
> > +      intstallation directory. If specified, the 'make install'
>
> s/intstallation/installation
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/snapcore/snapcraft/pull/715/files/3e16ff838ab22d27cbb20cc42c6a6f1ab44034e8#r74589626>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAYYdtrrWgNRDHYTaPK64zQetKk-xvOZks5qfHRjgaJpZM4Jfiqb>
> .
>